src/scene/scene.cpp

```diff
@@ -812,6 +812,16 @@ PropertyValueType Scene::propertyValueType(const std::string& value) {
     }
 }
 
+std::vector<properties::Property*> Scene::getPropertiesMatchingRegex(
```

Comment:
For the naming scheme, this should be called `propertiesMatchingRegEx` without the `get` part